fe/src/pages/admin/Category/Edit/EditCategory.jsx

```diff
@@ -13,32 +16,29 @@ function EditCategory({ id }) {
   return (
     <Fragment>
       <Form handler={handleSubmit}>
-        <ButtonModal
-          text={"Edit"}
-          style={"btn btn-outline-warning"}
-          target={"#EditCategoryModal"}
-          handler={handleGetCategory}
-        />
-
         <Modal
-          title={"Add New Category"}
-          modalID={"EditCategoryModal"}
-          formType={"update"}
+          title="Add New Category"
+          modalID="EditCategoryModal"
+          buttonText="Edit"
+          buttonStyle={OUTLINE_YELLOW}
+          formType={SOLID_BLUE}
+          submitText="Update Category"
+          handler={handleGetCategory}
         >
           <FormInput
-            label={"Title"}
-            name={"title"}
+            label="Title"
+            name="title"
             value={data.title || ""}
             handler={handleInput}
-            info={"Please enter lesson name"}
+            helper="Please enter lesson name"
```

Comment:
- its reverted back to `helper`. pls undo this one.
![image](https://user-images.githubusercontent.com/73863685/207247672-1090345c-a700-450f-b9f2-355af6b30ccc.png)